### PR TITLE
chore: more lp cleanup

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

- Prevent broken earnings query if no `from` timestamp is provided
- Fix file name typo
- Make `AddLiquidity` and `RemoveLiquidity` components lazy to prevent firing of effects in the background with incorrect data (I was seeing various mismatched calls for blockchain data with incorrect addresses - ex. /utxos with a thorchain address. This seemed to get rid of these scenarios, but may also be masking another issue?)
- Fix estimation memo (https://github.com/shapeshift/web/pull/6396 went in before this, but this ensures we always get a memo as long as we know the pool)
- Improve vernacular in `TransactionRow` and make sure we are using the correct `accountId` and `assetId` in each of the selectors.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

Low/Med - mostly cleanup, but does change some of the selectors so make sure everything still works

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- Ensure thorchain LP deposit and withdrawal for all types (sym, asym asset, and asym rune) still work as expected

### Engineering

:point_up: 

### Operations

:point_up:

### CODE REVIEW

**Review by 0xApotheosis — DISMISSED** [review]
Looking good, though found one regression where the "Sign transaction" button does not enter the loading state when pressed:

Screenshot 2024-03-12 at 3 40 22 PM

File: src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx [hunk 1]
```diff
@@ -674,11 +681,13 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
...
         <Amount.Crypto fontWeight='bold' value={amountCryptoPrecision} symbol={asset.symbol} />{' '}
         {isSymWithdraw && (
+          // Symmetrical withdrawals withdraw both asset amounts in a single TX.
+          // In this case, we want to show the pool asset amount in additional to the rune amount for the user
           <>
             <AssetIcon size='xs' assetId={poolAsset?.assetId} />
             <Amount.Crypto
               fontWeight='bold'
-              value={poolAmountCryptoPrecision}
+              value={confirmedQuote.assetWithdrawAmountCryptoPrecision}
```
Comment (gomesalexandre) [inline src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx:690]: 🧠

File: src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx [hunk 2]
```diff
@@ -708,11 +717,10 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
           <Button
             mx={-2}
             size='lg'
-            colorScheme={isTradingActive === false ? 'red' : 'blue'}
+            colorScheme={isTradingActive === false || status === TxStatus.Failed ? 'red' : 'blue'}
             onClick={handleSignTx}
-            isDisabled={isTradingActive === false}
+            isDisabled={isTradingActive === false || status === TxStatus.Failed}
             isLoading={
-              status === TxStatus.Pending ||
```
Comment (gomesalexandre) [inline src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx:715]: @kaladinlight this is the root cause of the regression, we want to bring this back

Comment (gomesalexandre) [reply src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx:715]: 1a323633b0

**Pull request merged**